dask/array/core.py

```diff
@@ -1314,6 +1314,16 @@ def cumprod(self, axis, dtype=None):
         from .reductions import cumprod
         return cumprod(self, axis, dtype)
 
+    def nancumsum(self, axis, dtype=None):
+        """ See da.nancumsum for docstring """
+        from .reductions import nancumsum
+        return nancumsum(self, axis, dtype)
+
+    def nancumprod(self, axis, dtype=None):
+        """ See da.nancumprod for docstring """
+        from .reductions import nancumprod
+        return nancumprod(self, axis, dtype)
```

Comment:
It looks like numpy doesn't add the nan-reductions as methods.  Not sure if we want to add them or not.  There was a bit of discussion on this after you and I ran into it together: https://github.com/dask/dask/issues/1045

I can go either way